src/shared/types/thoas/product.ts

```diff
@@ -43,6 +43,21 @@ export enum ProductType {
   PROTEIN = 'Protein'
 }
 
+type Location = {
+  start: number;
+  end: number;
+};
```

Comment:
The imported `LocationWithinRegion` was the correct type; no need to redefine it.

Reply:
Done